selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/DashboardWorkspace.java

```diff
@@ -628,4 +693,367 @@ public void waitWsTimeoutMessage(String expectedMessage) {
         .getText()
         .contains(expectedMessage);
   }
+
```

Comment:
Consider moving new methods into a new class.
Otherwise class becomes too huge.

Reply:
DashboardWorkspace page object will be reworked by https://github.com/eclipse/che/issues/6686.